app/agents/voice/breeze_buddy/workflows/order_confirmation/websocket_bot.py

```
@@ -494,6 +498,65 @@ async def _finalize_call(self):
+                    # Performance metrics
+                    if call_duration:
+                        self.root_span.set_attribute("call_duration_seconds", call_duration)
+                    if self.lead:
+                        self.root_span.set_attribute("attempt_count", self.lead.attempt_count + 1)
+                    
+                    # Try to fetch recording URL if available (may not be set yet)
+                    try:
+                        updated_lead = await get_lead_by_call_id(self.call_sid)
+                        if updated_lead and updated_lead.recording_url:
+                            self.root_span.set_attribute("recording_url", updated_lead.recording_url)
+                    except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)